### ISSUE TITLE
Adjusting autoHeight Slide when internal contents change

### ISSUE THREAD

**Issue by Inggo-inHomePortal** [smashingboxes/OwlCarousel2#1194]
I have a dynamic HTML content inside a slide that adjust heights (Bootstrap Accordions). Whenever the height changes, the contents get cut off if the original height of the container is too short.

My workaround is this:

    $(document).on('shown.bs.collapse hidden.bs.collapse', '.panel-collapse', function(e){
        $owl.trigger('next.owl.carousel', [0]).trigger('prev.owl.carousel', [0]);
    });

Basically, trigger a next/prev event if any accordion is clicked. But I feel that it's a bit hacky. Is there a proper way to do this? I tried refresh but I have no idea what event object to pass.
